_assignments/mp3.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: assignment
+title: "MP3: VR Critic"
+due: "March 4, 2019 @ 11:59PM"
+material: 
+  - name: Oculus Best Practices Guide
+    url: https://developer.oculus.com/design/latest/concepts/book-bp/
+points: 100
+notes: false
+instructions: false
+---
+
+You will review three VR experiences according to the [Oculus Best Practices Guide](https://developer.oculus.com/design/latest/concepts/book-bp/). These VR experiences can be from the Oculus Store, the Steam Store, or a student demo from the CS498 `V:` drive (in the "Student Projects" folder). There is no right or wrong answer. This assignment is meant to turn you into a VR snob; pick apart every little design decision you can think of.
+
+Each review must focus on the following four categories:
+1. Graphical/Aesthetic Quality
+1. User interface
+1. Movement
+1. Immersion/Presence
+
+For each category, write a short paragraph that describes what that particular VR experience does well and what it does poorly in regards to that category. Include at least two suggestions on how to improve the experience. Make sure to back up your critiques and suggestions with evidence from the Best Practices Guide. In total, write 150-300 words per experience.
+
+#### Grading:
+
+*You will be graded according to how well you can justify your answer with specific examples from the Oculus Best Practices Guide.*
+
+- Each review must cover all four categories. 
+- **YOU MUST PROVIDE PAGE NUMBER(S) FROM THE BEST PRACTICES GUIDE TO CITE A GIVEN OBSERVATION** 
+- **YOU MAY NOT CITE FROM THE FIRST NINE PAGES.** (these are just the overview)
+- Each review must include at least two suggested improvements that are consistent with the Best Practices Guide. 
+- Each of the three reviews counts for 1/3rd of your assignment grade.  
+- Reviews that do not fit the word limit will result in 20% deduction
+- Spelling and grammar mistakes will result in 10% deduction
+
+It is strongly encouraged that you make explicit reference to specific points from the Best Practices Guide whenever possible.  If you explain your ratings well, and it is clear you have read and understood the Best Practices Guide, then you will receive full credit. 
```

Comment:
"strongly encouraged to make explicit reference" and "must provide page numbers" conflict with each other.